src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVUAP.java

```diff
@@ -102,46 +111,42 @@ public void adsCheckSlotSizesOasis(String slotName, Page page) {
   @Test(
           dataProviderClass = AdsDataProvider.class,
           dataProvider = "adsVUAPTopDesktop",
-          groups = {"AdsVuapTimeProgressing", "AdsVuapTopOasis"}
+          groups = "AdsVuapTimeProgressing"
   )
-  public void adsVuapTimeProgressing(String slotName, Page page) throws InterruptedException {
-    int delay = 2;
-
+  public void adsVuapTimeProgressing(Page page, Map<String, String> map) throws InterruptedException {
     new AdsBaseObject(driver, urlBuilder.getUrlForPage(page), DESKTOP_SIZE);
-    VUAP vuap = new VUAP(driver, slotName);
+    VUAP vuap = new VUAP(driver, map.get(SRC), map.get(SLOT_NAME));
 
     vuap.play();
     vuap.waitForVideoStart();
-    Thread.sleep(delay * 1000);
+    Thread.sleep(DELAY * 1000);
```

Comment:
Consider using explicit wait instead of `Thread.sleep()` since the second isn't a good practice in Selenium world AFAIK. You can find a lot of explicit waits in our code, I believe. But maybe @sczerwinski-wikia, @nikodamn or @ludwikkazmierczak has some good advises in this area :)